controllers/articles.controller.js

```diff
@@ -21,8 +21,14 @@ exports.getArticles = (req, res) => {
 
 exports.getArticleComments = (req, res, next) => {
   const { article_id } = req.params;
-  fetchArticleComments(article_id)
-    .then((comments) => {
+  const promises = [fetchArticleComments(article_id)];
+
+  if (article_id) {
+    promises.push(fetchArticle(article_id));
+  }
+
```

Comment:
I have a feeling you might not need this if block? Worth playing around to see...there should always be an article_id right?